mobile-app/app/screens/TransactionAuthorization/PasscodePrompt.tsx

```diff
@@ -33,157 +36,135 @@ interface PasscodePromptProps {
   onModalCancel: () => void
   additionalMessage?: string
   additionalMessageUrl?: string
+  successMessage?: string
 }
 
 const PromptContent = React.memo((props: PasscodePromptProps): JSX.Element => {
   return (
     <>
-      <ThemedTouchableOpacity
-        dark={tailwind('bg-gray-900')}
-        light={tailwind('bg-white')}
+      <ThemedTouchableOpacityV2
+        light={tailwind('bg-mono-light-v2-100')}
+        dark={tailwind('bg-mono-dark-v2-100')}
         onPress={() => props.onCancel(USER_CANCELED)}
-        style={tailwind('items-end pt-2 pr-2')}
+        style={tailwind('items-end pt-6 pr-6 rounded-t-2xl')}
         testID='cancel_authorization'
         disabled={[TransactionStatus.BLOCK, TransactionStatus.SIGNING].includes(props.status)}
       >
-        <ThemedIcon
-          dark={tailwind('text-white')}
-          light={tailwind('text-black')}
-          iconType='MaterialIcons'
-          name='close'
-          size={26}
-        />
-      </ThemedTouchableOpacity>
-
-      <ThemedView
-        dark={tailwind('bg-gray-900')}
-        light={tailwind('bg-white')}
-        style={tailwind('w-full flex-1 flex-col')}
+        <ThemedViewV2
+          light={tailwind('border-mono-light-v2-900')}
+          dark={tailwind('border-mono-dark-v2-900')}
+          style={[tailwind('w-6 h-6 flex justify-center items-center rounded-full'), { borderWidth: 1.5 }]}
+        >
+          <ThemedIcon
```

Comment:
alignment is slightly off on iOS, looks ok on android, might need platform-specific styling, also can we change to 20px x 20px according to figma
![image](https://user-images.githubusercontent.com/40191153/178476815-151729e7-ff88-4dac-9548-c7e4a060c9d1.png)